fix: Typo in http/conditional_requests/index.md (#31614)
mdn/content: files/en-us/web/http/conditional_requests/index.md

Comparing versions of the same resource is a bit tricky: depending on the context, there are two kinds of _equality checks_:

- _Strong validation_ is used when byte to byte identity is expected, for example when resuming a download.

> - _Weak validation_ is used when the user-agent only needs to determine if the two resources have the same content. This is even if there are minor differences; like different ads, or a footer with a different date.

Reviewer: I think there's some more clarification we could add here while we're at it, what do you think?

```suggestion
- _Weak validation_ is used when the user-agent only needs to determine if two resources have the same content. The resources may be considered the same even if minor differences exist, such as different ads or a footer with a different date.
```

Author: Looks great, added to the PR. Thank you.

Reviewer: Super, tnx
